### DISCUSSION TITLE
How to install globally into the Program Files folder? Can the -g syntax be supported to install into the global directory instead of the user directory?

### DISCUSSION THREAD

**Discussion by tianlongJ** [microsoft/winget-cli#3777] [Q&A]
### Description of the new feature / enhancement

How to install globally into the Program Files folder? Can the -g syntax be supported to install into the global directory instead of the user directory?

**Comment by denelon** [comment]
If you are talking about using WinGet to install packages "machine" wide, you can use the "--scope machine" argument for package installers that support installing machine wide.

**Reply by tianlongJ** [reply]
It's too long, can it be simplified to -g?
